src/layouts/components/Header.tsx

```diff
@@ -179,9 +216,9 @@ function DesktopNavbar(): JSX.Element {
         />
         <HeaderLink
           className="ml-1 lg:ml-2"
```

Comment:
```suggestion
          className="ml-1 lg:ml-2 whitespace-nowrap"
```

header view breaking at 1440px
<img width="860" alt="image" src="https://user-images.githubusercontent.com/57183851/215372075-3b3c8b20-707d-4e38-9aaa-026c399e1ffa.png">